templates/ovs/bin/init.sh

```diff
@@ -13,20 +13,17 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
-set -x
+set -ex
 
 # Configs are obtained from ENV variables.
 OvnBridge=${OvnBridge:-"br-int"}
 OvnRemote=${OvnRemote:-"tcp:127.0.0.1:6642"}
 OvnEncapType=${OvnEncapType:-"geneve"}
-if [ -n "${OvnEncapNetwork}" ]; then
-  OvnEncapIP=$(/usr/local/bin/container-scripts/get_net_ip ${OvnEncapNetwork})
-else
-  OvnEncapIP=${OvnEncapIP:-"127.0.0.1"}
-fi
+OvnEncapNIC=${OvnEncapNIC:="eth0"} 
 EnableChassisAsGateway=${EnableChassisAsGateway:-false}
 PhysicalNetworks=${PhysicalNetworks:-""}
 OvnHostName=${OvnHostName:-""}
+OvnEncapIP=$(ip a s dev ${OvnEncapNIC} | grep "inet " | awk '{print $2}' | cut -d "/" -f 1)
```

Comment:
This does not work for IPv6.

Reply:
right, the current version won't support ipv6